src/main/java/org/broadinstitute/consent/http/models/DarCollectionSummary.java

```diff
@@ -251,6 +267,14 @@ public void addDacName(String dacName) {
     }
   }
 
+  public boolean getProgressReport() {
+    return progressReport;
+  }
+
+  private void updateProgressReportStatus() {
```

Comment:
Computed values that require an explicit update like this are hard to keep working correctly. Since gson still doesn't support getter-based properties, I can think of two approaches

- only generate these objects at the point before serialization, and have a generic "update computed fields" method which handles all cases. Or, change our code to use a builder pattern, and compute the fields at build time
- switch to using Jackson or some other JSON library for the cases where we want to do this

Reply:
I'd like to see if we can return to using Jackson but given current time constraints, I don't think that's possible in the context of this PR.